src/modules/validations.ts

```diff
@@ -14,8 +14,7 @@ import en from '../i18n/en-validation.json';
 yup.setLocale(en);
 
 // eslint-disable-next-line import/prefer-default-export
-export const ENS_DOMAIN_REGEX =
-  '^([A-Za-z0-9](?:[A-Za-z0-9-.]{0,255}[A-Za-z0-9])?)';
+export const ENS_DOMAIN_REGEX = '^[A-Za-z0-9][^.]{1,255}$';
```

Comment:
Arg! Apparently this is also used in `TextDecorator`, that's why this was set up with a capture group.

I'll fix it...

Reply:
So the only way I managed to properly solve this, is to have separate regexes for the ENS yup validator and the `TextDecorator` schema.

This is because the text decorator needs to match multiple ones, while the validator only matches the first.

Reply:
It would be really nice to have a comment just explaining why this regex is the way it is. Of course I am though perfectly literate in reading regex 😗 (this is a lie)